packages/template-cli/templates/deploy/azure/ssr/main.tf

```diff
@@ -20,7 +20,7 @@ variable "vnet_cidr" {
 }
 
 module "sample_aks_bootstrap" {
-  source                  = "git::https://github.com/amido/stacks-webapp-template//libs/orchestration/terraform-azurerm-amido-aks?ref=feat/1357"
+  source                  = "git::https://github.com/amido/stacks-webapp-template//libs/orchestration/terraform-azurerm-amido-aks?ref=feat/1435"
```

Comment:
Should this reference master?

Reply:
it should once we are ready to release it - it shold actually reference a release tag :) 
but for us for testing we can reference self  branch in monorepo